charts/corda-dev-prereqs/README.md

```diff
@@ -8,9 +8,10 @@ A Helm chart for installing Corda 5 development pre-requisites (Kafka and Postgr
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
-| imagePullSecrets | string | `nil` | List of secrets for pulling images |
+| imagePullSecrets | list | `[]` | List of secrets for pulling images |
```

Comment:
looks like the doc tool wasn't run after this change, but it's nothing to do with this PR